helpers/tile_variations.py

```diff
@@ -0,0 +1,77 @@
+import random
+from typing import Tuple
+
+
+def parse_variation_parameters(query: str) -> Tuple[str, str]:
+    """Parses 'variation' and parameters from a query string. Returns the remaining query string
+    as well as the variation value.
+
+    Args:
+        query: The query string to parse.
+    """
+    # main 'variation' use case (ex: ?tile flowers variation 3)
+    if 'variation' in query:
+        query, variation = [q.strip() for q in query.split('variation')]
+        if variation == '':
+            variation = 'random'
+    # support a simplified 'unidentified' query (ex: ?tile hypertractor unidentified)
+    elif query.endswith('unidentified'):
+        query = query.removesuffix('unidentified')
```

Comment:
`str.removesuffix` has not been added yet in Python 3.8 which the bot is currently running on